subworkflows/nf-neuro/tractometry_flow/meta.yml

```diff
@@ -0,0 +1,203 @@
+name: "tractometry_flow"
+description:
+  This workflow processes tractograms and associated data to clean, resample, and extract meaningful statistics.
+  It requires five input channels,a set of tractogram bundles, centroids used for resampling or identification,
+  various diffusion metrics associated with the tractography, a lesion probability mask and fixel-based FODF data.
+  The workflow performs key preprocessing and analysis steps such as invalid streamline removal, resampling,
+  label map creation, uniformization, and statistical extraction. The final output includes various tractography-related statistics, which can be used for further analysis.
+  You can retrieve intermediate outputs if you don’t want to run the entire subworkflow.
```

Comment:
Not sure about this sentence. You can retrieve intermediate outputs, but the subworkflow will run in its entirety nonetheless.